MMM-AlarmClock.js

```diff
@@ -73,6 +76,28 @@ Module.register('MMM-AlarmClock', {
         }
     },
 
+    fadeAlarm() {
+        var max_volume = this.config.volume;
+        var volume = 0;
+        var that = this;
+        var volume_step = this.config.fade_step;
+        var counter = 0;
+        var myTimer = setInterval(() => {
+            document.getElementById('alarm_player').volume = volume;
+            volume = volume + volume_step;
+            console.log(volume);
+            counter = counter + 1000;
+            if (volume >= max_volume) {
+                document.getElementById(that.identifier).volume = max_volume;
```

Comment:
the identifier of the module is not the same as the id you used in the other parts